cypress/e2e/git/branch_menu.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Test git branch menu
+
+  Background:
+    Given I clear cache
+    And I set viewport size to "1536" px for width and "960" px for height
+    And I visit the "/"
+
+    When I click on "[data-cy=\"create-empty-project\"]"
+    Then I expect current url is "/modelizer/project-[a-f0-9]{8}/model"
+    And  I extract "project-[a-f0-9]{8}" from url in field "projectName" of context
+    And I visit the "/#/modelizer/{{projectName}}/text"
+
+  Scenario: Default branch should be master
```

Comment:
no problem with `main` of GitHub ?

Reply:
no, because when you init in local, your default branch is master.